src/client/components/input/select-wrapper.js

```diff
@@ -86,7 +86,12 @@ class SelectWrapper extends React.Component {
 		const Child = base;
 
 		const childValue = _.isNil(value) ? this.state.value : value;
-
+		function getOptionLabel(option) {
```

Comment:
If you use `getOptionLabel = (option) => { ... }` instead, then `getOptionLabel={this.getOptionLabel}`, you don't get the react/jsx-no-bind warning